src/release.rs

```diff
@@ -628,7 +628,7 @@ static LLVM_URL: Lazy<Url> = Lazy::new(|| {
 /// Returns the path to the top-level `llvm` directory.
 pub async fn bootstrap_llvm() -> Result<PathBuf> {
     let url = &*LLVM_URL;
-    let filename = url.path_segments().unwrap().last().unwrap();
+    let filename = url.path_segments().unwrap().next_back().unwrap();
```

Comment:
"warning: called `Iterator::last` on a `DoubleEndedIterator`; this will needlessly iterate the entire iterator"

https://rust-lang.github.io/rust-clippy/master/index.html#double_ended_iterator_last